test/unit/alignment/pairwise/alignment_configurator_test.cpp

```diff
@@ -18,63 +18,49 @@ using namespace seqan3;
 auto setup()
 {
     auto data = std::tuple{"ACGT"_dna4, "ACGT"_dna4};
-    return ranges::view::single(std::move(data)) | view::persist;
+    return ranges::view::single(std::move(data));
 }
 
-TEST(alignment_configurator, configure_edit)
+template <typename config_t>
+bool run_test(config_t const & cfg)
 {
-    auto cfg = align_cfg::edit;
-
     auto r = setup();
     auto fn = detail::alignment_configurator::configure(r, cfg);
-    auto [seq1, seq2] = *seqan3::begin(r);
+    auto && [seq1, seq2] = *seqan3::begin(r);
```

Comment:
```suggestion
    auto && [seq1, seq2] = *std::ranges::begin(r);
```

Reply:
Woah, and I thought I did a regex replace on the entire subdirectory 😲

Reply:
I guessed so but I checked nonetheless :nerd_face:

Reply:
Oh it is in the tests, right 🤦‍♂️